### PR TITLE
catatonit: 0.1.7 -> 0.2.0

### PR DESCRIPTION
**Pull request by saschagrunert**
## Description of changes

Update to the latest release. 

cc @NixOS/podman @erosennin

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/virtualization/catatonit/default.nix
Comment (erosennin) [inline pkgs/applications/virtualization/catatonit/default.nix:1]: The project has been [relicensed from GPL-3.0-or-later to GPL-2.0-or-later](https://github.com/openSUSE/catatonit/releases/tag/v0.2.0), we should update the license accordingly.

**Review by erosennin — CHANGES_REQUESTED** [review]
The project has been [relicensed from GPL-3.0-or-later to GPL-2.0-or-later](https://github.com/openSUSE/catatonit/releases/tag/v0.2.0), we should update the license accordingly.

File: pkgs/applications/virtualization/catatonit/default.nix
Comment (saschagrunert) [reply pkgs/applications/virtualization/catatonit/default.nix:1]: Done, I think `gpl2Plus` is right?

Comment (erosennin) [reply pkgs/applications/virtualization/catatonit/default.nix:1]: Yes, seems right to me. :)

**Review by erosennin — APPROVED** [review]

**Review by erosennin — APPROVED** [review]

File: pkgs/applications/virtualization/catatonit/default.nix [hunk 1]
```diff
@@ -2,25 +2,15 @@
...
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-jX4fYC/rpfd3ro2UZ6OEu4kU5wpusOwmEVPWEjxwlW4=";
+    sha256 = "sha256-AqJURf4OrPHfTm5joA3oPXH4McE1k0ouvDXAF3jiwgk=";
   };
 
-  patches = [
-    # Pull the fix pending upstream inclusion to support automake-1.16.5:
-    #  https://github.com/openSUSE/catatonit/pull/18
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/virtualization/catatonit/default.nix:17]: You forgot to remove the unused fetchpatch from inputs. :)

Comment (saschagrunert) [reply pkgs/applications/virtualization/catatonit/default.nix:17]: Closing the loop: https://github.com/NixOS/nixpkgs/pull/258989

**Pull request merged**